templates/taxi/car_list.html

```diff
@@ -1,16 +1,33 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Car list</h1>
+  <h1>Car list <a href="{% url 'taxi:car-create' %}">+</a></h1>
   {% if car_list %}
-    <ul>
+    <table class="table">
+      <tr>
+        <th>ID</th>
+        <th>Name</th>
+        <th>Update</th>
+        <th>Delete</th>
+      </tr>
+    
       {% for car in car_list %}
-        <li>
-          <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
-          {{ car.model }} ({{ car.manufacturer.name }})
-        </li>
+        <tr>
+          <td>
+              <a href="{% url "taxi:car-detail" pk=car.id %} ">{{ car.id }}</a>
+          </td>
+          <td>
+              {{ car.model }} ({{ car.manufacturer.name }})
+          </td>
+          <td>
+            <a href="{% url 'taxi:car-update' pk=car.id %}">UPDATE</a>
+          </td>
+          <td>
+              <a href="{% url 'taxi:car-delete' pk=car.id %}">DELETE</a>
```

Comment:
Consider adding a 'Cancel' button next to the 'Delete' button to allow users to navigate back to the previous page. According to the checklist, a 'Cancel' button should be provided apart from the 'Delete' one.